content/doc/book/managing/system-configuration.adoc

```diff
@@ -11,5 +11,149 @@ ifdef::backend-html5[]
+Plugin support is provided for
+Subversion, Mercurial, Perforce, Team Foundation Server, and other SCMs.
+In most cases, the plugin for the tool must be installed separately
+before you configure the tool on the System Configuration page.
+
+== SCM and authentication
+
+Most SCM tools support or require authentication when talking to servers.
+Different SCMs are configured differently,
+but most (including Git, SVN, Mercurial, and Team Foundation Server)
+use the Credentials plugin.
+However, Perforce requires that each project
+enters credentials separately.
+////
+
+== Jenkins location
+
+You must configure the **Jenkins Location** area
+to identify this particular instance of Jenkins.
+
+Two items must be defined here:
+
+* *Jenkins URL* -- Replace the `localhost:5000` string with information
+about the actual location people can use to access this Jenkins instance.
+This information can be:
+
+** The actual URL that people use to access Jenkins.
+** The host alias as defined by the name service.
+If your IT people cannot provide this, use dynamic DNS.
+This makes your instance easier for your users to access
+and makes your service relocatable.
+
+[NOTE]
+====
+*Invest in a good URL!*
+If your users cannot see Jenkins, much of the benefit is lost.
+Make the URL easy to  remember.  For example:
+
+* Bad: `\http://tjek-02.int.rev.example.org`
+* Good: `\https://jenkins.example.org`
+====
+
+* *System Admin e-mail address* -- Define the email address to which email
+about general Jenkins issues are sent.
+This can be an individual's email address
+or an email alias that contains multiple email addresses.
+
+== Timestamper
+
+The **Timestamper** fields define the format used
+to display timestamp information.
+
+== More about system configuration
+
+Jenkins is very flexible and powerful.
+Default configurations can be used for most fields while you are getting started.
+As you learn more about how Jenkins works and how your project uses Jenkins,
+you can modify the values of settings on this page.
+Use these settings to fine-tune the behavior of your Jenkins instance.
+
+When you are done modifying this screen,
+scroll to the bottom and click
+*Save* to apply your configuration changes.
+
+== Management options
+
+Select **Managment Options** to start, stop, and reload the Jenkins service.
+
+To reload the configuration from disk,
+Jenkins rereads the configuration from `$JENKINS_HOME`
+without restarting Jenkins.
+
+The **Prepare for Shutdown** option
+prevents new jobs from being started
+and waits for running jobs to complete.
+Use this option to ensure a clean job termination before you shut down or restart Jenkins.
+
+The Management page provides Jenkins Upgrade management.
+If you are running a standalone server,
+you can perform a one-click upgrade.
+Otherwise, you have a direct link to the latest WAR.
+
+**Manage Old Data** helps to handle dropped configuration items after an upgrade.
+
+////
+== What did we learn?
+////
```

Comment:
```suggestion
Select **Managment Options** to start, stop, and reload the Jenkins service.

To reload the configuration from disk,
Jenkins rereads the configuration from `$JENKINS_HOME`
without restarting Jenkins.

The *Prepare for Shutdown* option prevents new jobs from being started, and waits for running jobs to complete.
Use this option to ensure a clean job termination before you shut down or restart Jenkins.

The Management page provides Jenkins Upgrade management.
If you are running a standalone server, you can perform a one-click upgrade.
Otherwise, you have a direct link to the latest WAR.

*Manage Old Data* helps to handle dropped configuration items after an upgrade.
```
small formatting and syntax suggestion, remove unneeded text